doc/plots.rst

```diff
@@ -0,0 +1,115 @@
+#####
+Plots
+#####
+
+The **Plots** pane shows the figures and images generated in the Editor, Console or the Variable Explorer during your session, allowing you to interact with them in several ways.
+
+.. image:: images/plots/plots-execution.gif
+   :align: center
+   :alt: Spyder main window showing execution for the Plots pane
+
+|
+
+The plots in the Plots pane depend on the Console you are working on. 
+This means that when switching to another console, the plots from your previous one will disappear, showing you a blank pane.
+
+.. image:: images/plots/plots-switch.gif
```

Comment:
This is both spatially large and temporally long for a GIF, producing a large 1.7 MB file. Switching between two consoles with existing plots would be much faster and simpler to show this, and you can reduce the spatial size by reducing the width to 500/480px and reducing the height of the panes. Or at the very least further speed up/cut out the various delays and intermediate steps not intrinsically linked with the core behavior behind demonstrated.

Reply:
> Switching between two consoles with existing plots would be much faster and simpler to show this

I agree. I also suggest that each console has an uneven number of plots on each one (e.g. two in one and 5 in the other), to highlight even more this point.

> and you can reduce the spatial size by reducing the width to 500/480px and reducing the height of the panes

I agree with this too.